lib/src/inline_syntaxes/autolink_extension_syntax.dart

```diff
@@ -13,7 +13,8 @@ class AutolinkExtensionSyntax extends InlineSyntax {
   static const _linkPattern =
       // Autolinks can only come at the beginning of a line, after whitespace,
       // or any of the delimiting characters *, _, ~, and (.
-      r'(?<=^|[\s*_~(>])'
+      // Note: Disable it, as Safari does not support lookarounds.
```

Comment:
For future archaeologists, we can expand on this comment a bit, like "Disable this piece for now, as... Consider re-enabling later."

Reply:
Fixed!